great_expectations/core/util.py

```diff
@@ -63,6 +63,14 @@ def convert_to_json_serializable(data):
     import numpy as np
     import pandas as pd
 
+    try:
+        import pyspark
+    except ImportError:
+        pyspark = None
+        logger.debug(
+            "Unable to load pyspark and pyspark.sql; install optional spark dependency for support."
```

Comment:
@Shinnnyshinshin Minor: The message leads one to believe that we are trying to import `pyspark.sql`, but it is not being imported (only `pyspark` on line 67).

Reply:
wonderful catch. made the change